### ISSUE TITLE
All MacOS builds broken "Previous version of google-api-python-client detected"

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#15600]
https://source.cloud.google.com/results/invocations/caae4b6b-6625-4492-b1e7-6092d0fb9a53/log

```
Traceback (most recent call last):
  File "tools/run_tests/run_tests.py", line 49, in <module>
    from python_utils.upload_test_results import upload_results_to_bq
  File "/Volumes/BuildData/tmpfs/src/github/grpc/tools/run_tests/python_utils/upload_test_results.py", line 29, in <module>
    import big_query_utils
  File "/Volumes/BuildData/tmpfs/src/github/grpc/tools/gcp/utils/big_query_utils.py", line 21, in <module>
    from apiclient import discovery
  File "/Users/kbuilder/Library/Python/2.7/lib/python/site-packages/apiclient/__init__.py", line 11, in <module>
    'Previous version of google-api-python-client detected; due to a '
RuntimeError: Previous version of google-api-python-client detected; due to a packaging issue, we cannot perform an in-place upgrade. To repair, remove and reinstall this package, along with oauth2client and uritemplate. One can do this with pip via
  pip install -I google-api-python-client
```

**Comment by jtattermusch** [comment]
CC @dgquintas  @adelez

**Comment by jtattermusch** [comment]
looks like google-api-python-client 1.7.0  was released 11 hours ago, which seems to coincide with the breakage
https://pypi.org/project/google-api-python-client/#history

**Comment by jtattermusch** [comment]
filed https://github.com/google/google-api-python-client/issues/516

**Comment by jtattermusch** [comment]
Temporarily fixed by #15602.
